scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java

```diff
@@ -174,6 +174,21 @@ public void destroy() {
         this.window.destroy();
     }
 
+    /**
+     * @return the configured Catalog url, if none has been specified, the default one is used instead
+     */
+    private String getCatalogUrl() {
```

Comment:
every time the window view is displayed ! unfortunately it can't be static as it is the SchedulerConfig (instance server-side) that gives me the right catalog url during runtime. I managed to set the catalogUrl only once so the method is only called once per page refresh.